net-libs/libhubbub/libhubbub-0.3.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic
+
+DESCRIPTION="HTML5 compliant parsing library, written in C"
+HOMEPAGE="https://www.netsurf-browser.org/projects/hubbub/"
+SRC_URI="https://download.netsurf-browser.org/libs/releases/${P}-src.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86 ~m68k-mint"
+IUSE="doc test"
+
+BDEPEND="virtual/pkgconfig
+	dev-util/netsurf-buildsystem
+	test? ( dev-lang/perl )"
+RDEPEND=">=dev-libs/libparserutils-0.2.1-r1"
+DEPEND="${RDEPEND}
+	test? ( dev-libs/json-c )"
+RESTRICT="!test? ( test )"
+
+DOCS=( README docs/{Architecture,Macros,Todo,Treebuilder,Updated} )
+PATCHES=( "${FILESDIR}/libhubbub-json-c.patch" )
```

Comment:
Your patch should include a version where it's required in/introduced in. Really just nitpicking, this ebuild looks good.

Reply:
Sorted